### PR TITLE
Migration v2

### PR DESCRIPTION
**Pull request by mmetayer**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     ],
     "require": {
         "akeneo/pim-community-dev": "2.*",
-        "akeneo-labs/custom-entity-bundle": "v2.0.2-beta",
+        "akeneo-labs/custom-entity-bundle": "2.0.*",
```
Comment (fitn) [inline composer.json]: 2.*?
We are already in 2.1 for this extension and I will release the 2.2 soon (with export fixes)

File: src/Resources/config/datagrid/brand.yml [hunk 2]
```diff
@@ -2,90 +2,101 @@ datagrid:
...
+                code:
+                    data_name: rd.code
+                label:
+                    data_name: rd.label
+                logo_url:
+                    data_name: rd.logoUrl
+            default:
+                code: '%oro_datagrid.extension.orm_sorter.class%::DIRECTION_ASC'
+
+        mass_actions:
```
Comment (fitn) [inline src/Resources/config/datagrid/brand.yml]: Why did you add these two mass actions on the datagrid?

Comment (mmetayer) [reply src/Resources/config/datagrid/brand.yml]: Because it was easy to do :), but I can remove them if you prefer
